src/mainwindow.cpp

```diff
@@ -147,8 +149,6 @@ MainWindow::MainWindow(QWidget *parent)
         settings.value(QStringLiteral("noteEditIsCentralWidget"), true)
             .toBool();
 
-    ui->setupUi(this);
```

Comment:
Will this break the "disable icons in the menu" part under macOS?

Reply:
I don't have a macOS to test that.

Reply:
Moved it to main.cpp instead. So, now it will be done before UI setup.